build/azure-pipelines/darwin/product-build-darwin.yml

```diff
@@ -59,6 +59,21 @@ steps:
     node build/lib/builtInExtensions.js
   displayName: Install distro dependencies and extensions
 
+- script: |
+    set -e
+    cd ..
+    git clone https://github.com/microsoft/vscode-telemetry-extractor.git
+    cd vscode-telemetry-extractor
+    npm install
+    ./node_modules/typescript/bin/tsc
+    npm run setup-extension-repos
+    node ./out/cli-extract.js --sourceDir ../s --excludedDirPattern extensions  --outputDir . --applyEndpoints --includeIsMeasurement --patchWebsiteEvents
+    node ./out/cli-extract-extensions.js --sourceDir ./src/telemetry-sources/ --outputDir . --applyEndpoints --includeIsMeasurement
+    mv declarations-resolved.json ../s/src/telemetry-core.json
+    mv declarations-extensions-resolved.json ../s/src/telemetry-extensions.json
+    echo 'Moved Files'
```

Comment:
This should just be removed.

Reply:
So Darwin doesn't need a specific build script?

Reply:
I mean the `echo`. :+1: